ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/console/components/build/clientlibs/js/build.js

```diff
@@ -22,7 +22,9 @@ $(function () {
         BUILD_IN_PROGRESS = 'BUILD_IN_PROGRESS',
         COMMAND_URL = Granite.HTTP.externalize("/bin/wcmcommand"),
         DIALOG_MODAL_URL = '/mnt/overlay/etoolbox-backpack/admin/console/page/content/editpackagedialog.html?packagePath=',
-        BUILD_PAGE_URL = '/tools/etoolbox/backpack/buildPackage.html?path=';
+        BUILD_PAGE_URL = '/tools/etoolbox/backpack/buildPackage.html?path=',
+        INSTALL = 'INSTALL',
+        INSTALL_IN_PROGRESS = 'INSTALL_IN_PROGRESS';
     $packageName = $('#packageName'),
```

Comment:
Looks like `var` is missing. The previous set of declarations ends with a semicolon

Reply:
And looks like `use strict` is missing as well) That's why you saw no warning. Please check that `use strict` is present in every JS file, unless there is a JS class or JS module which are strict-mode by default

Reply:
fixed, thanks